src/components/Menu/SocialLinksMenuButton.tsx

```diff
@@ -0,0 +1,77 @@
+"use client"
+
+import * as DropdownMenu from '@radix-ui/react-dropdown-menu'
+import {
+  Github,
+  Instagram,
+  Linkedin,
+  Rocket,
+  Twitter,
+  User,
+  Youtube
+} from 'lucide-react'
+import { MenuButton } from './MenuButton'
+
+export function SocialLinksMenuButton() {
+  const socialLinks = [
```

Comment:
As this doesn't change based on `props` or `state`, we can keep this outside the component function, what do you think?

Reply:
done. This is a really nice touch, I'm so used to put every variable inside, I will bring this to my routine.